index.js

```diff
@@ -42,8 +42,10 @@ const RARITIES = [
 ];
 client.on("message", function (message) {
   if (message.author.bot) return;
-  if (message.content.startsWith("!draw")) {
+  else if (message.content.startsWith("!draw")) {
```

Comment:
this works, but for clarity, it's better to keep this if statement separate from the one above. The intent of the first `if` is to filter out bots, the intent of the second `if` is to determine what command is being sent